act/io/arm.py

```diff
@@ -841,12 +841,12 @@ def read_arm_mmcr(filenames):
     # read it in with xarray
     multi_ds = []
     for f in filenames:
-        nc = Dataset(f, 'a')
-        # Change heights name to range to read appropriately to xarray
-        if 'heights' in nc.dimensions:
-            nc.renameDimension('heights', 'range')
+        nc = Dataset(f, 'r')
         if nc is not None:
```

Comment:
Is there a reason for this if check?

Reply:
@zssherman, I guess this was a legacy thing, so I didn't modify it.
Ideally, I'd rewrite that method using solely xarray and making it much shorter, but I do not have the time to do it at the moment, so in many aspects, this is more of a patch.